File: #7 Write Number in Expanded Form/main.js
function expandedForm(num) {
  str_num = String(num);
  arr = [];
  ind = 0;

  for (let i = 0; i < str_num.length; i++) {
    if (str_num[i] != 0) {
      arr.push(str_num[i] + "0".repeat(str_num.slice(ind + 1).length));
    }
    ind++;
  }
  return arr.join(" + ");
}

File: #7 Write Number in Expanded Form/main.py
def expanded_form(num):
    str_num = str(num)
    arr = []
    ind = 0

    for i in str_num:
        if i != "0":
          arr.append(i+ ("0" * len(str_num[(ind+1):])))
        ind+=1

    return " + ".join(arr)
